x-pack/plugins/security_solution/public/network/components/kpi_network/unique_private_ips/translations.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const NETWORK_EVENTS = i18n.translate(
+  'xpack.securitySolution.kpiNetwork.networkEvents.title',
+  {
+    defaultMessage: 'Network events',
+  }
+);
+
+export const UNIQUE_FLOW_IDS = i18n.translate(
+  'xpack.securitySolution.kpiNetwork.uniqueFlowIds.title',
+  {
+    defaultMessage: 'Unique flow IDs',
+  }
+);
+
+export const DNS_QUERIES = i18n.translate('xpack.securitySolution.kpiNetwork.dnsQueries.title', {
+  defaultMessage: 'DNS queries',
+});
+
+export const TLS_HANDSHAKES = i18n.translate(
+  'xpack.securitySolution.kpiNetwork.tlsHandshakes.title',
+  {
+    defaultMessage: 'TLS handshakes',
+  }
+);
+
+export const UNIQUE_PRIVATE_IPS = i18n.translate(
```

Comment:
I think we can just keep UNIQUE_PRIVATE_IPS here, and remove rest of them

Reply:
After we update the translation in each network kpi, I think we can also remove 
x-pack/plugins/security_solution/public/network/components/kpi_network/translations.ts